app/policies/organizations/invitation_policy.rb

```diff
@@ -3,6 +3,6 @@ class Organizations::InvitationPolicy < ApplicationPolicy
   pre_check :verify_active_staff!
 
   def create?
-    permission?(:invite_staff)
+    deny!
```

Comment:
This policy is setup to always deny where it is called. This is acceptable because the other `FostererInvitationPolicy` and `StaffInvtationPolicy` are used where necessary in the `InvitationsController`. This will also make sure anyone trying to go to the general invitations form page that is auto-generated by devise, will not be able to access the page.

The other actions such as edit are not authorized (since they mostly don't even have an authenticated user yet) and Action Policy is currently setup to not require authorization for Devise controllers.

Reply:
Makes sense!